x-pack/plugins/monitoring/common/constants.js

```diff
@@ -152,3 +152,10 @@ export const DEBOUNCE_FAST_MS = 10; // roughly how long it takes to render a fra
 export const CLUSTER_ALERTS_ADDRESS_CONFIG_KEY = 'cluster_alerts.email_notifications.email_address';
 
 export const STANDALONE_CLUSTER_CLUSTER_UUID = '__standalone_cluster__';
+
+export const INDEX_PATTERN = '.monitoring-*-6-*';
+export const INDEX_PATTERN_KIBANA = '.monitoring-kibana-6-*';
+export const INDEX_PATTERN_LOGSTASH = '.monitoring-logstash-6-*';
+export const INDEX_PATTERN_BEATS = '.monitoring-beats-6-*';
+export const INDEX_PATTERN_ALERTS = '.monitoring-alerts-6';
```

Comment:
This isn't a pattern. I'd just name this `INDEX_ALERTS`. FWIW, the setting was also named the same way (no "pattern").

Reply:
FINE HAVE IT YOUR WAY